README.md

```diff
@@ -22,6 +22,15 @@ process directly, or you can install the plugin as an npm module, include the
 javascript and SCSS source in your project using a Common-JS module loader and SASS build
 process, and copy the images from the image source folder to your project.
 
+Note that regardless of whether you are using this plugin via the pre-built JS or as a
+module, the Chromecast framework will need to be included after the plugin. For example:
+
+```html
+<script src="https://unpkg.com/video.js@6.1.0/dist/video.js"></script>
+<script src="../../dist/silvermine-videojs-chromecast.min.js"></script>
```

Comment:
Why the `../..`? Seems like this is copy/paste from somewhere. Maybe just replace `../..` with `.`?